docs/user-guide.md

```diff
@@ -24,26 +24,26 @@ that might be null then you write `@Nullable String x;`.
 
 ## Types and nullness
 
-In the JSpecify world, most references have one of three possible properties
+In the JSpecify world, most TODO references have one of three possible properties
 regarding nullness:
 1. They can be null.
 2. They can't be null.
 3. We don't know if they can be null or not.
 
-If `x` is a reference of a certain type, then if `x` can be null, `x.toString()` is
+If `x` is a reference of a certain type, then if `x` can be null, the dereference `x.toString()` is
```

Comment:
Maybe this is too nit-picky, but the guarantee is only about the dereference of `x` itself, not about what the method itself might do.

Reply:
I propose to change the example to `x.getClass()`. I think it will be clearer then that the only thing that can provoke NPE is if `x` is null.